StellarDotnetSdk.Xdr/ContractCostType.cs

```diff
@@ -42,8 +42,9 @@ namespace StellarDotnetSdk.Xdr;
 //      InvokeVmFunction = 13,
 //      // Cost of computing a keccak256 hash from bytes.
 //      ComputeKeccak256Hash = 14,
-//      // Cost of computing an ECDSA secp256k1 signature from bytes.
-//      ComputeEcdsaSecp256k1Sig = 15,
+//      // Cost of decoding an ECDSA signature computed from a 256-bit prime modulus
```

Comment:
I see updates here but should this not be used/updated anywhere?

Reply:
The whole file hasn't never been used anywhere actually.

Reply:
Does this mean we did something wrong before? Can you check if this is also never used in the Java SDK?

Reply:
I had checked both JS and Java SDK, never used.